### PR TITLE
A tail of Australian Shepherd tails and Harpy Wings (new sprites)

### PR DESCRIPTION
**Pull request by MortoSasye**
## About The Pull Request

Adds a new canid tail, inspired by Australian Shepherds with theirs still on them, with a wagging tail animation.

Adds a new set of harpy wings with a top overlay (which means it will behave like the options harpy top we have), that's been modified to allow most clothes to show on the arms without removing the feathers overlay.

## How This Contributes To The Skyrat Roleplay Experience

There was no similar canid tail in the options, and none of the harpy wings had a similar design to the one provided.

## Proof of Testing

<details>
<summary>Australian Shepherd Tail:</summary>
Sprite:
<br>
image

Tail Wagging:

https://github.com/Skyrat-SS13/Skyrat-tg/assets/38175176/3a67dcb8-1b1e-49b0-84d3-c19cea77adc2

</details>
<details>
<summary>Arfel Harpy Wings:</summary>
With no markings, and clothes on:
<br>
image
</details>

## Changelog

:cl:MortoSasye (Mystral Mom)
add: Two new sets of cosmetic sprites are now available: Arfel Harpy Wings and Australian Shepherd tail. Enjoy, new harpies and canids players.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/tails.dm [hunk 1]
```diff
@@ -420,3 +420,7 @@
 	name = "Slim reptile"
 	icon_state = "reptileslim"
 	color_src = USE_ONE_COLOR
+
+/datum/sprite_accessory/tails/mammal/wagging/australianshepherd
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/tails.dm]:
```suggestion
/datum/sprite_accessory/tails/mammal/wagging/australian_shepherd
```

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/wings.dm [hunk 2]
```diff
@@ -387,3 +387,8 @@
 /datum/sprite_accessory/wings/moth/moffra
 	name = "Moth (Moffra)"
 	icon_state = "moffra"
+
+/datum/sprite_accessory/wings/mammal/top/arfelharpy
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/wings.dm]:
```suggestion
/datum/sprite_accessory/wings/mammal/top/arfel_harpy
```

File: modular_skyrat/master_files/icons/mob/sprite_accessory/tails.dmi
Comment (GoldenAlpharex) [inline modular_skyrat/master_files/icons/mob/sprite_accessory/tails.dmi:1]: For this, the wag animation on the side views is far longer than the one from the front or the back. It also has a weird frame jump when it goes "behind" the character, where it goes from like, 45° to 90° in one frame, as opposed to being smoother like the rest.

As much as possible, animations should correspond between directions, _especially_ when it comes to the length/number of frames of the animation.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

File: modular_skyrat/master_files/icons/mob/sprite_accessory/tails.dmi
Comment (MortoSasye) [reply modular_skyrat/master_files/icons/mob/sprite_accessory/tails.dmi:1]: Got it, will fix it one of these days!

Comment (MortoSasye) [reply modular_skyrat/master_files/icons/mob/sprite_accessory/tails.dmi:1]: Changes done! I also took advantage of the situation and adjusted the shading of everything.

**Review by vinylspiders — DISMISSED** [review]
Couple things:

1) Make sure you doc the Shark tail changes and Akula tail removal. And speaking of the second, if you are removing the sprites you'll want to get rid of this too (or else the 'Akula' tail selection in char prefs will lead to an invisible tail).

https://github.com/Skyrat-SS13/Skyrat-tg/blob/78795c33c7417d3268b307fe3c5b0271472640e2/modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/tails.dm#L312-L314

2) The sprites look good to me but with

`m_waggingtail_shark_BEHIND_primary` and `m_waggingtail_sharknofin_BEHIND_primary`

the modified shark tail sprites still have some abrupt frame shifts. edit: I realize those were there with the originals too so if you can't fix it that's okay. But if you want to:

See what I mean <details><summary> here (you can see the skipping -- 1st and 3rd ones down)</summary> 

firefox_wUuVSQZGMC 

</details>

Definitely address 1) though, and if you know how to fix the screenshot test that'd be great as well but if not I'll take care of it.

**Comment by MortoSasye** [comment]
> Couple things:
> 
> 1. Make sure you doc the Shark tail changes and Akula tail removal. And speaking of the second, if you are removing the sprites you'll want to get rid of this too (or else the 'Akula' tail selection in char prefs will lead to an invisible tail).
> 
> https://github.com/Skyrat-SS13/Skyrat-tg/blob/78795c33c7417d3268b307fe3c5b0271472640e2/modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/tails.dm#L312-L314
> 
> 2. The sprites look good to me but with
> 
> `m_waggingtail_shark_BEHIND_primary` and `m_waggingtail_sharknofin_BEHIND_primary`
> 
> the modified shark tail sprites still have some abrupt frame shifts. edit: I realize those were there with the originals too so if you can't fix it that's okay. But if you want to:
> 
> See what I mean
> 
> here (you can see the skipping -- 1st and 3rd ones down)
> Definitely address 1) though, and if you know how to fix the screenshot test that'd be great as well but if not I'll take care of it.

I didn't even notice I removed anything...

Edit: I don't see the changes at all myself, nor did my husband. I'm very new to this whole coding business though so maybe I didn't even find out I did the edit in a wrong file? I asked for help over at the discord channel too.

**Comment by vinylspiders** [comment]
> > Couple things:
> > 
> > 1. Make sure you doc the Shark tail changes and Akula tail removal. And speaking of the second, if you are removing the sprites you'll want to get rid of this too (or else the 'Akula' tail selection in char prefs will lead to an invisible tail).
> > 
> > https://github.com/Skyrat-SS13/Skyrat-tg/blob/78795c33c7417d3268b307fe3c5b0271472640e2/modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/tails.dm#L312-L314
> > 
> > 
> > 2. The sprites look good to me but with
> > 
> > `m_waggingtail_shark_BEHIND_primary` and `m_waggingtail_sharknofin_BEHIND_primary`
> > the modified shark tail sprites still have some abrupt frame shifts. edit: I realize those were there with the originals too so if you can't fix it that's okay. But if you want to:
> > See what I mean
> > here (you can see the skipping -- 1st and 3rd ones down)
> > Definitely address 1) though, and if you know how to fix the screenshot test that'd be great as well but if not I'll take care of it.
> 
> I didn't even notice I removed anything...
> 
> Edit: I don't see the changes at all myself, nor did my husband. I'm very new to this whole coding business though so maybe I didn't even find out I did the edit in a wrong file? I asked for help over at the discord channel too.

Ah was a misunderstanding then! We spoke on discord, but to reiterate all you have to do is update your tails.dmi to have the missing Akula tail sprites that got added after this PR was started and this should be good to go as far as I can tell.

**Review by GoldenAlpharex — APPROVED** [review]
Yeah sure, looks good to me now.

**Pull request merged**